java/libphonenumber/src/com/google/i18n/phonenumbers/MetadataManager.java

```diff
@@ -61,19 +61,15 @@
   private MetadataManager() {
   }
 
-  private static void close(InputStream in) {
-    if (in != null) {
-      try {
-        in.close();
-      } catch (IOException e) {
-        LOGGER.log(Level.WARNING, e.toString());
-      }
-    }
-  }
+  // The size of the byte buffer in bytes used to convert a stream containing metadata for a single
+  // region, to a nanoproto-compatible CodedInputByteBufferNano. This was determined by the size of
+  // the metadata with which we build our actual jars.
```

Comment:
I believe @dlyongemallo wrote the numbers this way to signify that 1024 bytes is a Kb, so this is 16 Kb (power-of-2 ceiling of the largest per-region binary metadata file with room for growth), and the other is 256 Kb (power-of-2 ceiling of the single binary metadata file containing all regions' metadata, with room for growth - we don't support clients using this - it's experimental - and we don't ship this file to Maven Central).